shoes-swt/lib/shoes/swt/button.rb

```diff
@@ -11,6 +11,11 @@ def initialize(dsl, app)
           button.set_text @dsl.text
         end
       end
+
+      def text=(value)
+        @last_text = @real.text
```

Comment:
Any particular reason you're holding onto this here? I don't see other uses of `@last_text` anywhere else in button-related code.